src/_data/sites/teladoc-health.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "Teladoc Health",
+	"url": "https://www.teladochealth.com",
+	"description": "Teladoc Health offers clients a broad spectrum of high-quality virtual care solutions to meet the healthcare needs of patients on a global scale.",
+	"twitter": "teladochealth",
+	"authoredBy": ["araphiel"],
```

Comment:
@araphiel Is this a Twitter handle? Per the [template](https://github.com/11ty/11ty-website/issues/new?assignees=&labels=new-site%21&template=built-with-eleventy.md&title=I+built+a+site+with+Eleventy%21):
> * _`authoredBy`: An array of Twitter usernames of the site’s authors. Supplements the `twitter` entry. (Optional)_

I tried checking Twitter but it says:

> **Account suspended**
> Twitter suspends accounts which violate the Twitter Rules. Learn more.